homeassistant/components/hassio/ingress.py

```diff
@@ -109,6 +109,9 @@ async def _handle(
     delete = _handle
     patch = _handle
     options = _handle
+    connect = _handle
```

Comment:
The documentation of `CONNECT` at MDN reads:

> If a website is behind a proxy and it's enforced via network rules that all external traffic must pass through the proxy, the CONNECT method allows you to establish a [TLS](https://developer.mozilla.org/en-US/docs/Glossary/TLS) ([HTTPS](https://developer.mozilla.org/en-US/docs/Glossary/HTTPS)) connection with that website:
> ...
>  * The proxy server makes a secure connection to the server on behalf of the client.

I'd rather prefer if we don't implement this if the current implementation really doesn't do what `CONNECT` should.

Reply:
deleted